docs/wallets/app-wallet/transfer-hotspot.mdx

```diff
@@ -98,18 +96,14 @@ No. Once transferred, the Hotspot and its mining rewards moving forward belong t
 
 The Seller pays for the transaction fee.
 
-### Is this available on the CLI?
-
-Yes. Wallet CLI is available for this feature.
-[Helium CLI Wallet Documentation](/wallets/cli-wallet)
 
 ### Will this be recorded in the blockchain?
 
 Yes. Transfer Hotspot transactions are on-chain and visible in Explorer.
 
 ### What if the activity check reveals the Hotspot hasn’t done anything recently?
 
-You won’t be able to transfer the Hotspot.
+You won’t be able to transfer the Hotspot. Use the Wallet app to do this.
```

Comment:
There are no checks on activity now, right?

Reply:
correct, removing this item